whisper/config.yaml

```diff
@@ -12,6 +12,7 @@ discovery:
   - wyoming
 backup_exclude:
   - "*.bin"
```

Comment:
@synesthesiam can this be removed? You added this in #3116 but it doesn't seem to do anything, at least for the model I'm using.

Reply:
I tried many different models and I didn't see any bin files so I removed this line. The new pattern seems to capture all the models. I forgot to delete all the unused the models after testing and my last backup is 4 GB.

Reply:
I changed the source of the models in a previous version, but forgot to update the backup exclusion. Thank you for this.